src/env.js

```diff
@@ -145,6 +145,26 @@ const envDefinitions = {
     desc:
       "gasLimit * gasPrice to determine the fees price. A too low GAS_PRICE will get rejected before the transaction is broadcast",
   },
+  CRYPTO_ORG_INDEXER: {
+    def: "https://crypto.org/explorer",
+    parser: stringParser,
+    desc: "location of the crypto.org indexer API",
+  },
+  CRYPTO_ORG_TESTNET_INDEXER: {
+    def: "https://crypto.org/explorer/croeseid3",
+    parser: stringParser,
+    desc: "location of the crypto.org indexer testnet API",
+  },
+  CRYPTO_ORG_RPC_URL: {
+    def: "",
```

Comment:
Missing value here.
```
$ ledger-live sync -c crypto_org
Expected string `endpoint` to be an url, got ``
```

Reply:
yes please only use crypto_org_croeseid for now. Mainnet node is still being build but because they are overloaded with tasks, it might takes month for them to setup

Reply:
Noted :+1: